change/@fluentui-react-charting-51f43c57-5f52-424a-a747-874776dcb240.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29817&commitId=75a34446b451eab719011f2ed7bb74782a2a3ce2&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-SparkLineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SparkLineChart.Basic - RTL.default.chromium.png|436| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
The PR fixes the text overflowing issue as shown in the original screenshot.